Cargo.toml

```diff
@@ -22,15 +22,22 @@ frame-system-rpc-runtime-api = { path = "pallets/system/rpc/runtime-api" }
 frame-system-benchmarking = { path = "pallets/system/benchmarking" }
 
 # DA Primitives
-da-primitives = { version = "0.4", git="https://github.com/availproject/avail-core", tag = "da-primitives/v0.4.7" }
-kate = { version = "0.7", git="https://github.com/availproject/avail-core", tag = "da-primitives/v0.4.7" }
-kate-recovery = { version = "0.8", git="https://github.com/availproject/avail-core", tag = "da-primitives/v0.4.7" }
+# avail-core = { version = "0.5", git="https://github.com/availproject/avail-core", tag = "avail-core/v0.4.7" }
```

Comment:
This deps will be updated after the companion PR in avail core: https://github.com/availproject/avail-core/pull/27